src/index.html

```diff
@@ -5,9 +5,268 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="icon" href="../src/images/favicon.png">
+    <link href="https://fonts.googleapis.com/css2?family=Ubuntu:ital,wght@0,300;0,400;1,400;1,500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="body">
+    <div class="header">
+      <div class="header__call-bar">
+        <div class="header__logo logo" id="header-logo"></div>
+        <div class="header__phone">
+          <a class="header__link" href="tel:+1 234 555-55-55">
+            +1 234 555-55-55
+          </a>
+        </div>
+        <div class="header__order">
+          <a href="tel:+1 234 555-55-55" class="header__link">
+            call to order
+          </a>
+        </div>
+      </div>
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item nav__item-active">
+            <a class="nav__link-active" href="header-logo">
+              home
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#about" class="nav__link">
+              about us
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#gallery" class="nav__link">
+              gallery
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#consultation" class="nav__link">
+              consultation
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#testimonials" class="nav__link">
+              testimonials
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#contacts" class="nav__link">
+              contacts
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </div>
+
+    <section class="section" id="about">
+      <div class="section__size info">
+        <div class="info__heading">
+          miami <strong>best</strong> real estate<a class="info__link" href="#"> here</a>
+        </div>
+        <div class="info__text">
+          <p><strong>Over 50 years experience in love.</strong>Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.</p>
```

Comment:
```suggestion
          <p>
            <strong>Over 50 years experience in love.</strong>Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.
          </p>
```

Старайся форматувати так щоб було максимально приємно читати. Переносів багато не буває)